editor/store/defaults.js

```diff
@@ -1,9 +1,10 @@
 export const PREFERENCES_DEFAULTS = {
-	mode: 'visual',
-	sidebars: {
-		desktop: true,
-		mobile: false,
-		publish: false,
+	editorMode: 'visual',
```

Comment:
Given that we are changing the property name from mode to editorMode and also changing the selector we also need to update the preferences reducer on line 547.

Reply:
Why change the property name?

Reply:
@aduth @jorgefilipecosta I renamed `viewMode` to `viewportType`